scripts/vaccination_alerts/generate-vaccine-alerts-info.ts

```diff
@@ -4,33 +4,58 @@ import {
   getCmsVaccinationInfo,
   getFirebaseVaccinationInfo,
   getUpdatedVaccinationInfo,
+  DEFAULT_ALERTS_FILE_PATH,
 } from './utils';
 
+import * as yargs from 'yargs';
+
 /**
  * Generates the `vaccination-alerts.json` file with a map of locations that
  * have updated vaccination information.
  *
  * Run via: yarn vaccinations-generate-alerts
  */
 
-const alertsFilePath = path.join(__dirname, 'vaccination-alerts.json');
-
-async function main() {
+async function main(alertsFilePath: string) {
+  console.log('Loading latest alerts from cms data');
   const cmsInfo = getCmsVaccinationInfo();
+
+  console.log('Reading latest alerts sent from Firebase');
   const firebaseInfo = await getFirebaseVaccinationInfo();
 
+  console.log('Getting snapshot of alerts to send');
   const vaccinationAlertUpdates = getUpdatedVaccinationInfo(
     cmsInfo,
     firebaseInfo,
   );
 
-  fs.writeFileSync(alertsFilePath, JSON.stringify(vaccinationAlertUpdates));
+  for (const fips of Object.keys(vaccinationAlertUpdates)) {
```

Comment:
nit: consider using `const [fips, current] of Object.entries(vaccinationAlertUpdates)`